code_challenges/Account_Generator.py

```diff
@@ -0,0 +1,60 @@
+import random
```

Comment:
The file name should be lower_snake_case.py.

Reply:
fixed.